install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -584,6 +584,12 @@ cni:
   # inside the agent pod.
   hostConfDirMountPath: /host/etc/cni/net.d
 
+  # -- Specifies the resources for the cni container
```

Comment:
minor nit: please call this the "cni initContainer"

Reply:
Another request: the top-level `cni` grouping is for broader CNI settings, rather than individual agent DaemonSet bits. I'd rather see this as part of the agent daemonset configuration.

Unfortunately, we don't have a super clean way of doing that, since the agent settings are top-level, rather than being grouped under something.

So, I'd rather see the key be `initContainerResources.install-cni`.

Reply:
Updated to `cni initContainer`, sorry for the lag, I completely missed this

Reply:
So you want:
```
cni:
    ....
initContainerResources:
    install-cni:
        enabled: true
        resources:
        ....
```

Is that an accurate description? A few questions:
- Do we break compatibility with the existing options? If not how do we navigate the world with the two options 😬 ?
- Do I also migrate the other init containers?

Reply:
(sorry, inbox got a bit ahead of me)

That was my idea. However, if you think the cni resources are causing more harm than good, then I'd be OK with removing them. The only thing the container does is a few `cp`s.

Also, a cleaner hierarchy would probably be

```yaml
initContainers:
  install-cni-binaries:
    resources:
      ...
```

Thoughts?

Reply:
@RenaudWasTaken were you able to apply this feedback? Are you waiting on @squeed ?

Reply:
Sorry about the lag, the main thought was how do we manage backwards compatibility?

Reply:
cc @squeed